lti_consumer/utils.py

```diff
@@ -13,6 +13,13 @@ def _(text):
     return text
 
 
+def lti_1p3_enabled():
+    """
+    Returns `true` if LTI 1.3 integration is enabled for course.
+    """
+    return settings.FEATURES.get('LTI_1P3_ENABLED', False) is True
```

Comment:
I think this should be documented as well.

Reply:
I've added a note about this in the `installation` section of the README file.